Source/Blazorise/ComponentActivator.cs

```diff
@@ -48,6 +61,34 @@ public IComponent CreateInstance( Type componentType )
             return component;
         }
 
+        /// <summary>
+        /// Loads the ServiceProvider's list of disposables
+        /// </summary>
+        /// <returns>List of object references the ServiceProvider uses to track disposables</returns>
+        private IList<object> LoadServiceProviderDisposableList()
+        {
+            var disposablesPropertyInfo = ServiceProvider.GetType()
```

Comment:
@David-Moreira, I think we could optimize this by compiling it to an expression.

Reply:
Hello,
Damn this PR is exactly what we were missing on Blazorise. I tought there would be no workaround if there wasn't a fix provided implicitly by .NET.

Can't wait to try it out. I should be able to tomorrow, and I'll see if the expression makes sense.
(PS: I've seen articles a few months ago with comparisions that reflection has come a long way in recent dotnet versions and that the old knowledge we have that reflection is hella slow, is no longer 100% true, matter of testing I guess... ^^)

Reply:
So this `LoadServiceProviderDisposableList` is only ever called once per client, I don't think there's any noticeable performance hit. Right?

Reply:
It is called once if the service is registered as `Scoped`. If it is `Transient` then it will be called every time for every component.

Reply:
Kk then.
tested & modified.
![image](https://user-images.githubusercontent.com/22283161/190865683-b0801c8c-b6b7-4a01-9877-4bd79ac5e621.png)

Reply:
The results speak for themselves. I'm impressed by how much of a difference it is.